hooks/tk-nuke_scene_operations.py

```diff
@@ -53,6 +76,7 @@ def scan_scene(self):
             path = node.knob('file').value().replace("/", os.path.sep)
 
             reads.append( {"node": node_name, "type": "Read", "path": path})
+            logger.debug("inside my code, i can log like thiszzzzz")
```

Comment:
remove?

Reply:
Err...yeah...didn't catch that in the original PR.